src/testCaseEditor.ts

```diff
@@ -236,6 +259,37 @@ function atdToCatala(tests: TestList, lang: string): string {
   }
 }
 
+function runTestScope(filename: string, testScope: string): TestRunResults {
+  /*
+   * Notes:
+   * - when parsing / generating tests, we operate on the current text buffer
+   * in the editor through `stdin`. Here, we run the actual file on disk.
```

Comment:
I think we want to run the current text buffer and not the file stored on disk, this will create a lot of confusion otherwise...

Reply:
Possibly but I'm unsure of how other test runners (pytest...) work when invoked from an IDE. That would be worth checking